l10n/en/products/vpn/platforms/linux_new.ftl

```diff
@@ -0,0 +1,39 @@
+vpn-linux-mozilla-vpn-is-a = { -brand-name-mozilla-vpn } is a virtual private network service that uses open-source state-of-the-art encryption and does not log, track or share any of your network activity. It allows you to connect to over 500 servers in 30+ countries.
+vpn-linux-why-choose = Why choose { -brand-name-mozilla-vpn } for { -brand-name-linux }?
+vpn-linux-fast-and = <strong>Fast and reliable:</strong> { -brand-name-mozilla-vpn } uses the { -brand-name-wireguard }® protocol, which offers better performance and stability than other VPN protocols
+vpn-linux-no-logs = <strong>No logs:</strong> { -brand-name-mozilla-vpn } does not keep any logs of your network activity; we don’t record which websites you visit or inspect your traffic.
+vpn-linux-additional-security = <strong>Additional security features:</strong> We use DNS blocking to block ads, trackers, and malware. While a browser can prevent only websites from giving you malware and tracking you.
+vpn-linux-device-level = <strong>Device-level encryption:</strong> Unlike VPNs that only encrypt your browser traffic, { -brand-name-mozilla-vpn } encrypts all your device’s traffic, including all your applications and games.
+vpn-linux-multi-hop = <strong>Multi-hop routing:</strong> For added protection, { -brand-name-mozilla-vpn } can encrypt your traffic and route it through two locations instead of one, making it harder for anyone to trace your online activity.
+vpn-linux-simple-and = <strong>Simple and intuitive interface:</strong> Connect to a server with one click. You can also customize yoursettings and preferences according to your needs.
+vpn-linux-it-helps-restore = <strong>It helps restore net neutrality:</strong> { -brand-name-mozilla-vpn } helps you take back control of your online activities by preventing internet providers from throttling your bandwidth, or from blocking websites or services that they don’t like, or that compete with their own offerings.
+vpn-linux-gui-client = <strong>GUI client:</strong>{ -brand-name-mozilla-vpn } has a GUI client (graphical user interface), whereas some VPN providers only provide a CLI version (command line).
+vpn-linux-by-using = By using { -brand-name-mozilla-vpn }, you can take back control of your online activities and protect your privacy and data. Download { -brand-name-mozilla-vpn } today and enjoy a 30-day money-back guarantee with no logs, no hassle, and no risk.
+vpn-linux-how-to-install = How to install { -brand-name-mozilla-vpn } on { -brand-name-linux }
+
+# Variables:
+#   $url (string) - https://launchpad.net/~mozillacorp/+archive/ubuntu/mozillavpn
+# 'mozillavpn' should not be translated as it is the propername of the package
+vpn-linux-if-you-use = If you use one of the supported {-brand-name-ubuntu } releases, there are two ways to install mozillavpn official packages hosted on <a href="{ $url }" rel="external noopener noreferrer"target="_blank">launchpad:</a>
```

Comment:
Should be a space between attributes:
```suggestion
vpn-linux-if-you-use = If you use one of the supported {-brand-name-ubuntu } releases, there are two ways to install mozillavpn official packages hosted on <a href="{ $url }" rel="external noopener noreferrer" target="_blank">launchpad:</a>
```
Or, for this and the other links with extra attributes, it might be better to move all of that to the template instead of the string. The string could just have `<a { $attrs }>launchpad:</a>`